docs/UserGuide.md

```diff
@@ -348,11 +351,11 @@ will display the guests that already have their rooms cleaned.
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about using `find`:**<br>
-* The search is **case-insensitive**. E.g., `hans` will match `Hans`
-* The order of the keywords does not matter. E.g., `Hans Bo` will match `Bo Hans`
-* **Only full words** will be matched E.g., `Han` will not match `Hans`
-* Guests matching **at least one** keyword will be returned (i.e. `OR` search).
-  E.g., `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* The search is **case-insensitive**. E.g., _hans_ will match _Hans_.
+* The order of the keywords does not matter. E.g., _Hans Bo_ will match _Bo Hans_.
```

Comment:
Not sure if we should use search terms here (in Line 342 as well) rather than keywords, but in a sense keywords might be more layman for the user to understanding. What do you think? If you change to keywords, need to change Line 342 as well!

Reply:
I think we used search term everywhere else too, so maybe we should standardise and use search terms!

Reply:
I think we should stick to keyword because the command format uses the keyword term, might be more understandable with we standardise